lib/facter/resolvers/disk.rb

```diff
@@ -9,7 +9,13 @@ class Disk < BaseResolver
         init_resolver
 
         DIR = '/sys/block'
-        FILE_PATHS = { model: 'device/model', size: 'size', vendor: 'device/vendor', type: 'queue/rotational' }.freeze
+        FILE_PATHS = { model: 'device/model',
+                       size: 'size',
+                       vendor: 'device/vendor',
+                       type: 'queue/rotational',
+                       sn: 'false',
+                       wwid: 'false'
```

Comment:
Is there any reason you're not just reading the values from their respective files?

WWID can also be read from `/sys/block/<disk>/wwid`, so you'd just have to put `wwid` here instead of `false`.
SN on my system can be read from `/sys/block/<disk>/device/serial` so you'd have to put `device/serial` instead of `false`.

Reply:
I‘ve tested it with rotational disks and SSDs on centos7 and there were no files that give me the values reliable. 

So I‘ve put false in there to circle around the problem.